shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-sqlserver/src/main/antlr4/imports/sqlserver/BaseRule.g4

```diff
@@ -422,3 +427,7 @@ ignoredIdentifiers
 matchNone
     : 'Default does not match anything'
     ;
+
+variableName
+    : AT_ identifier
+    ;
```

Comment:
@Icesource There should be a empty line at the end of the file.

Reply:
@jingshanglu done